app.py

```diff
@@ -34,7 +37,28 @@ def eip_subgraph_info():
         "collectionDailySnapshots",
         hosted=False,
     )
+    
+#     hardcoded_query = """query {
+#   collectionDailySnapshots(where: 
+#     {collection: "0x34d85c9cdeb23fa97cb08333b511ac86e1c4e258"}, 
+#     first: 1, orderBy: dailyTradeVolumeETH, orderDirection: desc) {
+#     blockNumber
+#     dailyTradeVolumeETH
+#     timestamp
+#   }
+# }"""
+    # data = query_thegraph("AwoxEZbiWLvv6e3QdvdMZw4WDURdGbvPfHmZRc8Dpfz9",
+    #     hardcoded_query,
+    #     "collectionDailySnapshots",
+    #     hosted=False,)
```

Comment:
we need `data` in order for the app to function, needs to be uncommented and reverted to using `openai_result`